paper.md

```diff
@@ -233,6 +233,18 @@ This allowed DataLad and many other projects to facilitate access to an ever-gro
 
 # Overview of the DataLad and its ecosystem
 
+## Design principles
+
+The development of DataLad is guided by a few fundamental principles to ensure its open and domain agnostic nature, to maximize the longterm utility of its datasets and to minimze the technical debt introduced by using it:
+
+- There are only two recognized entities: datasets and files
+- A dataset is a Git repository with an optional annex
+- Minimization of custom procedures and data structures: Users must not loose data or data access, if DataLad would vanish
+- Complete decentralization, no required central server or service
+
+In that sense DataLad can be regarded as a thin wrapper around Git and git-annex, since -- technically -- everything DataLad can do could be done by using Git/git-annex directly and any modification of a dataset by DataLad remains accessible/recoverable by Git/git-annex alone.
```

Comment:
Maybe we can try to avoid the phrase "thin wrapper", because on a superficial level is may read somewhat closely to what the review criteria of JOSS define to be non-exceptable submissions: 

> Minor 'utility' packages, including 'thin' API clients, and single-function packages are not acceptable.

Reply:
Yeah - I'm not happy with the wording of that sentence either. However, the point seems important to me. Suggestions highly welcome ;-)

Reply:
How about replacing the paragraph after the bullet points with something like this:

"Due to these principles, using DataLad does not introduce a technical dependency of a user's data on DataLad or DataLad-related services as any modification of a dataset by DataLad remains accessible/recoverable by Git/git-annex alone."

This maybe circumvents the need to call datalad a thin wrapper. The point "While it can be a rather complex task to actually do any of that without DataLad" would be lost, but taken up (at least partially) by the addition in #80

Reply:
Yeah, prob. enough. Done.